### PR TITLE
Fix layout of alternative login buttons

### LINKED ISSUE #6367: Modernize style of 'Alternative Logins' buttons
Apps can add their own 'Login' mechanism by using [`\OC_App::registerLogIn`](https://github.com/nextcloud/server/blob/d7d2d7c803b372278cd60a7af76e6a410dce046d/lib/private/legacy/app.php#L718) which displays a new button on the initial login page (see the following screenshot):

nextcloud-alternative-login

These buttons have probably not been redesigned for years 🙀 

### How to reproduce

Install and enable the [Registration](https://github.com/pellaeon/registration) app.

### PR DESCRIPTION
**Pull request by MorrisJobke**
* fixes #6367
* multiple buttons will appear below

bildschirmfoto 2017-09-05 um 18 39 45
bildschirmfoto 2017-09-05 um 18 40 19
bildschirmfoto 2017-09-05 um 18 42 40

### CODE REVIEW

**Comment by MorrisJobke** [comment]
The last screenshot is slightly out of date, because it is not that black anymore. It's adjusted now to the grey value of the login button. 😉

**Comment by leonklingele-work** [comment]
@MorrisJobke this was fast 👍 
The screenshots look awesome, will test the patch tomorrow.

**Review by skjnldsv — APPROVED** [review]
Good for me! 👍

File: core/css/guest.css [hunk 1]
```diff
@@ -400,8 +400,23 @@ form .warning input[type='checkbox']+label {
...
 
 /* Alternative Logins */
-#alternative-logins legend { margin-bottom:10px; }
-#alternative-logins li { height:40px; display:inline-block; white-space:nowrap; }
+#alternative-logins legend {
+	margin-bottom: 10px;
+}
+#alternative-logins li {
+	height: 40px;
+	white-space:nowrap;
```
Comment (leonklingele-work) [inline core/css/guest.css]: Missing space after colon

File: apps/theming/css/theming.scss [hunk 2]
```diff
@@ -31,13 +31,15 @@
...
+		#alternative-logins li a {
 			border: 1px solid nc-lighten($color-primary-text, 50%);
 		}
-		input.primary {
+		input.primary,
+		#alternative-logins li a {
 			background-color: $color-primary;
 		}
-		a, label, p {
+		a, label, p, #alternative-logins legend {
```
Comment (leonklingele-work) [inline apps/theming/css/theming.scss]: Any reason you didn't put `#alternative-logins` into its own line as done above and below?

**Review by leonklingele-work — APPROVED** [review]

File: apps/theming/css/theming.scss (hunk 2 above)
Comment (MorrisJobke) [reply apps/theming/css/theming.scss]: Was just to leave it as it was styled before. Let me update the styles.

**Pull request merged**